src/chat/participant.ts

```diff
@@ -117,7 +124,8 @@ async function getModel(selector: LanguageModelChatSelector): Promise<LanguageMo
   let models: LanguageModelChat[] = [];
   for (const fieldToRemove of ["id", "version", "family", "vendor"]) {
     models = await lm.selectChatModels(selector);
-    logger.debug(`${models.length} available chat model(s)`, { models, modelSelector: selector });
+    // NOTE: uncomment for local debugging; this can be noisy otherwise
+    // logger.debug(`${models.length} available chat model(s)`, { models, modelSelector: selector });
```

Comment:
Only useful for inspecting the available properties of the full model listing, primarily for future updates if/when we start getting indicators that certain models support participants/tools/etc